JS-108_javascript-basics/code-challenge/combine-and-manipulate-strings/app.js

```diff
@@ -0,0 +1,8 @@
+// Challenge: Combine and Manipulate Strings
+// Task 1: Assign your first name to the variable firstName and your last name to lastName.
+let firstName = 'Paul Eliczar';
+let lastName = 'Adraneda';
+let role = 'developer';
```

Comment:
I didn't see the need to use `let`. The variables were not updated after all. Change `let` to `const`.
```suggestion
const firstName = 'Paul Eliczar';
const lastName = 'Adraneda';
const role = 'developer';
```

Reply:
@xianvalencia Right, I overlooked this part. I've commit the changes. 👍